src/commcare_cloud/ansible/roles/backups/tasks/main.yml

```diff
@@ -60,6 +60,21 @@
     - cron
     - backups
 
+- name: Create Hourly Cron job
+  become: yes
+  cron:
+    name: "Backup blobdb hourly"
+    job: "/usr/local/sbin/create_blobdb_backup.sh hourly {{ blobdb_backup_hours * 60 }}"
+    minute: 15
+    hour: "*/{{ nadir_hour_frequency|default(3) }}"
```

Comment:
where is `nadir_hour_frequency` defined?

Reply:
@snopoke I have not defined the variable explicitly but I have used a default value if not defined. However, we can define the variable either in public.yml/defaults file which gets overwritten. Please let me know if you want me to define it explictly. Thanks.

Reply:
I think you should define it in `all.yml` with a comment explaining what it is

Reply:
@snopoke sure. Will do that. Thanks.

Reply:
@snopoke added. please review Simon. Thanks.